src/components/DepositWidget/useRowActions.tsx

```diff
@@ -100,10 +100,16 @@ export const useRowActions = (params: Params): Result => {
       const receipt = await depositApi.deposit({ userAddress, tokenAddress, amount }, txOptionalParams)
       log(`The transaction has been mined: ${receipt.transactionHash}`)
 
-      _updateToken(tokenAddress, ({ depositingBalance, walletBalance, ...otherParams }) => {
+      _updateToken(tokenAddress, ({ pendingDeposit, walletBalance, totalExchangeBalance, ...otherParams }) => {
```

Comment:
I'm thinking about removing this and letting the network/new block update the state instead of doing it "by hand".
@anxolin @Velenir @W3stside thoughts?

Reply:
Yes, I'm very much in favor of one source of truth -- blockchain

Reply:
https://github.com/gnosis/dex-react/pull/349